felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -456,6 +458,7 @@ func newBPFEndpointManager(
 	featureDetector environment.FeatureDetectorIface,
 	healthAggregator *health.HealthAggregator,
 	dataplanefeatures *environment.Features,
+	hostMTU int,
```

Comment:
Called podMTU one later up, maybe bring them into sync or make this `bpfIfaceMTU` so it's more clear what it's for?